nservicebus/rabbitmq/configuration-api.md

```diff
@@ -14,197 +14,242 @@ To use RabbitMQ as the underlying transport:
 
 snippet:rabbitmq-config-basic
 
-The transport needs to connect the the RabbitMQ broker. By default the transport will look for a connection string called `NServiceBus/Transport` in the `app.config`.
+
+## Specifying the connection string
+
+The transport needs to connect to a RabbitMQ broker. By default, the transport will look for a connection string called `NServiceBus/Transport` in the `app.config`.
+
+To use a custom name for the connection string:
+
+snippet:rabbitmq-config-connectionstringname
+
+To specify the connection string in code:
+
+snippet:rabbitmq-config-connectionstring-in-code
 
 A typical connection string would look like this:
 
 snippet:rabbitmq-connectionstring
 
-In the above sample the transport is configured to connect to the RabbitMQ broker running at the machine `broker1`.
+In the above sample, the transport is configured to connect to a RabbitMQ broker running at the machine `broker1`.
+
+
+### Connection string options
 
 Below is the list of connection string options. Note that they are separated by a `;`.
 
 
+#### Host
+
+The host name of the broker.
+
+NOTE: This value is required.
+
+
 #### Port
 
-The port where the broker listens. 
+The port where the broker listens.
 
-Defaults: `5672`
+Default: `5672` or `5671` if `UseTls` is true
 
 
 #### VirtualHost
 
 The [virtual host](https://www.rabbitmq.com/access-control.html) to use.
 
-Defaults: `/`
+Default: `/`
 
 
 #### UserName
 
-The username when connecting. 
+The user name to use to connect to the broker.
 
-Defaults: `guest`
+Default: `guest`
 
 
 #### Password
 
-The password when connecting. 
+The password to use to connect to the broker.
 
-Defaults: `guest`
+Default: `guest`
 
 
 #### RequestedHeartbeat
 
 The interval for the heartbeats between the client and the server.
 
-Defaults: `5` seconds
+Default: `5` seconds
 
 
 #### DequeueTimeout
 
 The time period allowed for the dequeue strategy to dequeue a message.
 
-Defaults: `1` second
+Default: `1` second
 
 Versions: 3 and below.
 
 
 #### PrefetchCount
 
-The number of messages to [prefetch](http://www.rabbitmq.com/consumer-prefetch.html) when consuming messages from the broker. 
+The number of messages to [prefetch](http://www.rabbitmq.com/consumer-prefetch.html) when consuming messages from the broker.
 
-Defaults: The number of configured threads for the transport(as of v2.1)
+Default: The number of configured threads for the transport(as of v2.1)
 
-Versions: 3 and below. In Versions 4 and above `EndpointConfiguration.LimitMessageProcessingConcurrencyTo` can be used instead. See [Throughput Throttling](/samples/throttling/) sample.
+Versions: 3 and below.
```

Comment:
How about:

Versions: This setting is only available for Versions 3 and below. Use [what setting replaces this?] for Versions 4 and above.